model/database/metrics.yaml

```
@@ -133,3 +134,12 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.query.response.item_count
```

nit:

I wonder if it should be 
```suggestion
  - id: metric.db.response.item_count
```

since we have `db.response.status_code`.

I'll bring it up on the DB semconv tomorrow